src/utils/index.js

```diff
@@ -10,3 +13,16 @@ export async function promiseMap (array, fn) {
   const resolved = await Promise.all(promiseArray)
   return resolved
 }
+
+export function formatDateTime (isoDate) {
```

Comment:
I'm not sure this belongs in utils as its potentially pretty specific to the component you're using it in. Let's keep it in the component until we find that we need it somewhere else.